src/layouts/Map.styled.jsx

```diff
@@ -156,7 +157,7 @@ const FooterMapAddress = styled.div`
 
   @media (min-width: 960px) {
     font-size: 1em;
-    padding: 3.78% 0;
+    padding: 2% 0;
```

Comment:
Please avoid percentage like a plague. Please don't introduce new relative padding/margin/height, even width should avoid relative sizes when possible. If you introduce relative sizes please be sure to add stop values (`min-`/`max-`).